addons/hr_holidays/report/hr_leave_reports.xml

```diff
@@ -110,17 +110,14 @@
         </field>
     </record>
 
-    <record id="act_hr_employee_holiday_request" model="ir.actions.act_window">
+    <record id="act_hr_employee_holiday_request" model="ir.actions.server">
         <field name="name">Time off Analysis</field>
-        <field name="type">ir.actions.act_window</field>
-        <field name="res_model">hr.leave.report</field>
+        <field name="model_id" ref="hr.model_hr_employee"/>
```

Comment:
I'd define the method on the report model instead of the employee

```suggestion
        <field name="model_id" ref="hr.model_hr_leave_report"/>
```